openapi_server/openapi/openapi.yaml

```diff
@@ -119,6 +119,39 @@ paths:
       - sample
       x-openapi-router-controller: openapi_server.controllers.sample_controller
   /samples/{id}:
+    delete:
+      description: Delete a sample.
+      operationId: samples_id_delete
+      parameters:
+      - explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          $ref: '#/components/schemas/SampleID'
+        style: simple
+      responses:
+        "200":
```

Comment:
I would suggest to use `204` and leave the response body empty as there will no such entry if the `DELETE` has been successfully acted on.

Reply:
We would also need to update the spec @giang-nghg